device-types/Cisco/IE-2000-8TC-B.yaml

```diff
@@ -0,0 +1,33 @@
+---
+manufacturer: Cisco
+model: IE-2000-8TC-B
+part_number: IE-2000-8TC-B
+slug: cisco-ie-2000-8tc-b
+u_height: 3
+is_full_depth: false
+interfaces:
+  - name: FastEthernet1/1
+    type: 100base-tx
+  - name: FastEthernet1/2
+    type: 100base-tx
+  - name: FastEthernet1/3
+    type: 100base-tx
+  - name: FastEthernet1/4
+    type: 100base-tx
+  - name: FastEthernet1/5
+    type: 100base-tx
+  - name: FastEthernet1/6
+    type: 100base-tx
+  - name: FastEthernet1/7
+    type: 100base-tx
+  - name: FastEthernet1/8
+    type: 100base-tx
+  - name: FastEthernet1/9
+    type: 100base-tx
+  - name: FastEthernet1/10
+    type: 100base-tx
+console-ports:
+  - name: con 0
+    type: rj-45
+  - name: usb
+    type: usb-mini-b
```

Comment:
Is this used for console access? If not, please remove it from this file.

Reply:
used for console